templates/default/customer/customerassignments.html

```diff
@@ -44,6 +44,19 @@
 				<input type="checkbox" class="commited" value="1"{if $commited} checked{/if}>
 			</label>
 			<a class="clear-filter lms-ui-button" href="#"><i class="lms-ui-icon-delete"></i></a>
+			{button type="link" icon="tiphelp" tip="<strong>{trans("Assignments colors markings")}:</strong>
+			<br><br>
+			<span style='color:maroon'><strong>{trans("Active tariff assignment")}</strong></span>
```

Comment:
Fajnie by było, że to było klasą CSS, a ne stylem inline.

Reply:
Linia 49?

Reply:
Tak. Wypadałobyu to "uklasowić" np.